core/src/test/java/org/apache/calcite/test/SqlHintsConverterTest.java

```diff
@@ -112,12 +113,14 @@ protected DiffRepository getDiffRepos() {
     sql(sql).ok();
   }
 
-  @Test public void testInvalidQueryHint() {
+  @Test public void testInvalidQueryHint1() {
     final String sql = "select /*+ weird_hint */ empno\n"
```

Comment:
Thanks for the refactoring, can we give the test cases more meaningful names ?

Reply:
make sense.